third_party/tonic/dart_state.h

```diff
@@ -49,6 +49,11 @@ class DartState : public std::enable_shared_from_this<DartState> {
   Dart_Isolate isolate() { return isolate_; }
   void SetIsolate(Dart_Isolate isolate);
 
+  // TODO(dnfield): delete this https://github.com/flutter/flutter/issues/50997
```

Comment:
I don't believe we are getting that from Dart. This workaround is the way to go for speed.

Reply:
My thought is to repurpose the bug for tracking knocking this down - since it has all the context.